specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -278,37 +277,36 @@ finally, commit the transaction.
 
 === Transaction Interoperability
 
-=== Multiple Java EE Platform Interoperability
+=== Multiple Jakarta EE Platform Interoperability
 
 This specification does not require the Product
 Provider to implement any particular protocol for transaction
-interoperability across multiple Java EE products. Java EE compatibility
-requires neither interoperability among identical Java EE products from
-the same Product Provider, nor among heterogeneous Java EE products from
+interoperability across multiple Jakarta EE products. Jakarta EE compatibility
+requires neither interoperability among identical Jakarta EE products from
+the same Product Provider, nor among heterogeneous Jakarta EE products from
 multiple Product Providers.
 
-We recommend that Java EE Product Providers use
+We recommend that Jakarta EE Product Providers use
 the IIOP transaction propagation protocol defined by OMG and described
-in the OTS specification (and implemented by the Java Transaction
-Service), for transaction interoperability when using the EJB
+in the OTS specification (and implemented by the Jakarta Transactions
```

Comment:
@bshannon how is JTS being handled in the transition to Jakarta?

Reply:
JTS was never a JCP spec.  My understanding is that it is "just" a Java mapping to OTS.
The entire implementation is included in the GlassFish ORB.

The simplest thing is probably just to remove the reference to "Java Transaction Service".
Definitely do NOT change it to "Jakarta Transaction Service".

Reply:
Removed reference to Java Transaction Service.